jacodb-panda-dynamic/src/main/kotlin/org/jacodb/panda/dynamic/ets/dto/Model.kt

```diff
@@ -0,0 +1,140 @@
+/*
+ *  Copyright 2022 UnitTestBot contributors (utbot.org)
+ * <p>
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ * <p>
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.jacodb.panda.dynamic.ets.dto
+
+import kotlinx.serialization.ExperimentalSerializationApi
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.decodeFromStream
+import java.io.InputStream
+
+@Serializable
+data class EtsFileDto(
+    val name: String,
+    val absoluteFilePath: String? = null,
+    val projectDir: String? = null,
+    val projectName: String? = null,
+    val namespaces: List<NamespaceDto>,
+    val classes: List<ClassDto>,
+    val importInfos: List<ImportInfoDto>,
+    val exportInfos: List<ExportInfoDto>,
+) {
+    companion object {
+        private val json = Json {
+            // classDiscriminator = "_"
```

Comment:
Do we need it?

Reply:
No, we have `@JsonClassDiscriminator("_")` annotations on all DTOs. I was not sure whether they work correctly, so I kept `classDiscriminator = "_"` setting, as we commonly used it before annotations. The annotations seem to be working correctly, so we can drop these commented settings.